tests/integration/sentinel/transitions/registration.spec.js

```diff
@@ -98,6 +98,150 @@ describe('registration', () => {
   after(() => utils.revertDb([], true));
   afterEach(() => utils.revertDb(getIds(contacts), true));
 
+  it('should add valid phone to patient doc', () => {
+    const patient_phone = '+9779841123123';
```

Comment:
Please use camelCase for variables. 
```suggestion
const patientPhone = '+9779841123123';
```